schemas/context/enduserids.schema.json

```diff
@@ -9,42 +9,30 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "End User IDs",
   "type": "object",
-  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.",
-  "minProperties": 1,
```

Comment:
If at least one of the properties is required, why do we remove the formalized requirement?

Reply:
Not sure why that was remove. @jwen-adobe, is there a reason for this? 

Yes, at least on property is required.

Reply:
@cmathis @trieloff The original requirement was at least one field is required, but we can not call out which field..Other ways we can do for this case?

Reply:
@jwen-adobe - Isn't that what the "minProperties" is saying. You need to have at least one of the fields populated?

Reply:
@cmathis Yes we can use the minProperties here. Was thinking of oneOf and do not want to use it.